### ISSUE TITLE
initialCommands causes console task to hang in scala-2.9.1

### ISSUE THREAD

**Issue by josharnold52** [sbt/sbt#192]
I think scala-2.9.1 broke something in the `initialCommands` functionality.    It seems that any use of `initialCommands` causes the REPL to hang after its first prompt.

I can reproduce the issue with the following minimal `build.sbt`:

```
scalaVersion := "2.9.1"

initialCommands in console := "val testValue  = List(1,2,3)"
```

Here's my sbt-session

```
[info] Set current project to default-464d80 (in build file:/Users/arnold/projects/sbt-bug/)
> console
[info] Starting scala interpreter...
[info] 
testValue: List[Int] = List(1, 2, 3)
Welcome to Scala version 2.9.1.final (Java HotSpot(TM) 64-Bit Server VM, Java 1.6.0_26).
Type in expressions to have them evaluated.
Type :help for more information.

scala> :help
```

The console hangs at that point until I control-C out of it.

The problem goes away if I remove the `initialCommands` or use an earlier version of scala.

I did find a new issue logged against the interpreter that might be related, but I haven't had any luck applying the suggested workaround this context:

https://issues.scala-lang.org/browse/SI-4945

**Comment by josharnold52** [comment]
This is a duplicate of #175; Sorry about that.

**Comment by josharnold52** [comment]
I actually found a relatively simple (if ugly) fix for the console task that appears to work across all scala versions.  I know #175 was marked wontfix, but is there any chance of reconsidering?

see: josharnold52/xsbt@ed5afb44e53471167be18e97622b795f4b18d9e1

I'm not wild about about swallowing the NoSuchMethodException (needed to run against pre-2.9.1 versions), but maybe there's an easy way to explicitly check for the presence of the method instead.

Note that I coded against 0.10 accidentally.  I can easily make a 0.11 version if desired.
